cosign.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: cosign
+  version: 1.13.1
+  description: Container Signing
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - apk-tools

```

Comment:
We shouldn't need this at runtime. I think we will need CA certs at runtime though.